pkgs/cli/src/node.zig

```diff
@@ -191,8 +195,12 @@ pub const Node = struct {
     }
 
     fn constructMultiaddrs(self: *Self) !struct { listen_addresses: []const Multiaddr, connect_peers: []const Multiaddr } {
-        const self_node_index = self.options.validator_indices[0];
-        try ENR.decodeTxtInto(&self.enr, self.options.bootnodes[self_node_index]);
+        if (std.mem.eql(u8, self.options.validator_config, "genesis_bootnode")) {
+            try ENR.decodeTxtInto(&self.enr, self.options.bootnodes[self.options.node_key_index]);
+        } else {
+            // parse from validator_config and not from nodes.yaml
+            return error.NotImplemented;
+        }
```

Comment:
@bomanaps can you create the correct ENR here from parsing from `parsed_validator_config` rather than picking and loading from nodes.yaml because as you see if the node is node a bootnode (or e.g. I want to start a sync node) then its ENR  needs to be constructed

So we correctly construct ENR from `parsed_validator_config`'s `enrFields` itself in the first place